.github/scripts/ep_hooks.py

```
@@ -0,0 +1,267 @@
+        (context_dir / "pr-meta.json").write_text(
+            json.dumps(ticket, indent=2, default=str)
+        )
+
+    # ── Prompt builder ──
+
+    def build_prompt(self, ticket_key, mode, skill_name, **kw):
+        if skill_name == "prd-review":
+            return self._prd_prompt()
+        return self._design_prompt()
+
+    def _prd_prompt(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_prd_prompt`

Add return type annotation: `str`

(ANN202)